### PR TITLE
17992 - NR(LLP, LP, XLP) link to paper form

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:*
https://github.com/bcgov/entity/issues/17992

*Description of changes:*
NR (LLP, LP, XLP) will link to paper form

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]
LGTM. Nice job integrating into the existing code!

**Comment by severinbeauvais** [comment]
@kzdev420 What account did you use for testing? (So that I don't have to create 3 new NRs :) )

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

**Review by seeker25 — APPROVED** [review]
Looks great to me, nice job. 

Edit: Need to get CI passing before Squash + Merge please

**Review by chenhongjing — APPROVED** [review]
LGTM 🐱

**Comment by severinbeauvais** [comment]
@kzdev420 You need to work with Patrick to get the form download working in the temp URL.

image

**Comment by kzdev420** [comment]
> @kzdev420 What account did you use for testing? (So that I don't have to create 3 new NRs :) )
`

> Details

Hi @severinbeauvais
do you mean this `X @google-cloud-build - account-ui-ci-pr (yfthig-dev) — Summary`
I don't think I have an account for testing

**Comment by kzdev420** [comment]
> @kzdev420 You need to work with Patrick to get the form download working in the temp URL.
> 
> image

Hi @pwei1018 

I want you to help me with this.

**Comment by severinbeauvais** [comment]
> Hi @severinbeauvais do you mean this `X @google-cloud-build - account-ui-ci-pr (yfthig-dev) — Summary` I don't think I have an account for testing

Not this but Patrick can help you with configuring the new keys so that the code works in the temp URL.

**Comment by severinbeauvais** [comment]
@kzdev420 Actually, the form links probably didn't work because the 1Password (vault) keys did not exist. I just created them according to your .env.example so it might work now (except for the current Openshift issue).

**Comment by kzdev420** [comment]
> Actually, the form links probably didn't work because the 1Password (vault) keys did not exist. I just created them according to your .env.example so it might work now (except for the current Openshift issue).

Aha, Sorry. I forgot to leave the comments about vault keys. Thanks so much @severinbeauvais

**Comment by severinbeauvais** [comment]
Kevin, please test that the form links work before merging this PR. Thx.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

**Comment by severinbeauvais** [comment]
Kevin, I just redeployed the temp URL, and I'm still seeing this error when I click Download Form:

image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

File: auth-web/devops/vaults.env [hunk 1]
```diff
@@ -12,6 +12,9 @@ VUE_APP_APPROVE_ACCOUNT_SLA_DAYS="op://relationship/$APP_ENV/auth-web/APPROVE_AC
 VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
 VUE_APP_SOCIETIES_URL="op://web-url/$APP_ENV/bcregistry/SOCIETIES_URL"
 VUE_APP_CORP_FORMS_URL="op://web-url/$APP_ENV/bcregistry/CORP_FORMS_URL"
+VUE_APP_LLP_FORMS_URL="op://web-url/$APP_ENV/bcregistry/LLP_FORMS_URL"
+VUE_APP_LP_FORMS_URL="op://web-url/$APP_ENV/bcregistry/LP_FORMS_URL"
+VUE_APP_XLP_FORMS_URL="op://web-url/$APP_ENV/bcregistry/XLP_FORMS_URL"
```
Comment (severinbeauvais) [inline auth-web/devops/vaults.env:17]: Please move these 5 to the web-url section below.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2591-8n2fqak9.web.app

**Comment by severinbeauvais** [comment]
All - we know the forms don't work in the preview build but they do work locally, so I'm going to merge this PR and we will try it in Dev. It might be broken in Dev, but it's already broken in Dev (ie, this ticket). We will see.

**Pull request merged**